extensions/ql-vscode/src/remote-queries/shared/variant-analysis.ts

```diff
@@ -72,10 +72,13 @@ export interface VariantAnalysisSkippedRepositories {
 
 export interface VariantAnalysisSkippedRepositoryGroup {
   repositoryCount: number,
-  repositories: Array<{
-    id?: number,
-    fullName: string
-  }>
+  repositories: Array<VariantAnalysisSkippedRepository>,
```

Comment:
I think the `Array` syntax makes sense for more complicated types, but for simple arrays I think it's easier to read it as a simple array:

```suggestion
  repositories: VariantAnalysisSkippedRepository[],
```

See also this TypeScript ESLint rule: https://typescript-eslint.io/rules/array-type/#array-simple

Reply:
Thanks. Sounds good to me 👍